test/WebJobs.Script.Tests.Integration/Storage/BlobStorageConcurrencyStatusRepositoryTests.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Azure.WebJobs.Script;
+using Microsoft.Azure.WebJobs.Script.Tests;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.WebJobs.Script.Tests;
+using Moq;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Host.EndToEndTests
+{
+    /// <summary>
+    /// TODO: TEMP - this temporary implementation should be removed once https://github.com/Azure/azure-webjobs-sdk/issues/2710 is addressed.
```

Comment:
Copied all the tests over from [here](https://github.com/Azure/azure-webjobs-sdk/blob/33eb24a6a91facb5399dd59f50a363616826f4bf/test/Microsoft.Azure.WebJobs.Host.EndToEndTests/BlobStorageConcurrencyStatusRepositoryTests.cs#L22)